### PR TITLE
Running gofmt on the sources

### PR DESCRIPTION
**Pull request by vdemeester**
Produced via: `gofmt -s -w $(find -name '*.go' | grep -v vendor)`

Doing the same as the sockpuppet :joy_cat:  knative/build#475

/cc @mattmoor @bobcatfish 

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

### CODE REVIEW

File: pkg/reconciler/v1alpha1/pipelinerun/resources/providedby_test.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ var mytask1 = &v1alpha1.Task{
 	Spec: v1alpha1.TaskSpec{
 		Inputs: &v1alpha1.Inputs{
 			Resources: []v1alpha1.TaskResource{
-				v1alpha1.TaskResource{
+				{
```
Comment (mattmoor) [inline pkg/reconciler/v1alpha1/pipelinerun/resources/providedby_test.go]: You can collapse this onto the previous line to reduce indent (also below)

File: pkg/reconciler/v1alpha1/taskrun/resources/input_resource_test.go [hunk 2]
```diff
@@ -138,7 +138,7 @@ func TestAddResourceToBuild(t *testing.T) {
 		Spec: v1alpha1.TaskSpec{
 			Inputs: &v1alpha1.Inputs{
 				Resources: []v1alpha1.TaskResource{
-					v1alpha1.TaskResource{
+					{
```
Comment (mattmoor) [inline pkg/reconciler/v1alpha1/taskrun/resources/input_resource_test.go]: FWIW there's a bunch of these.

**Comment by vdemeester** [comment]
@mattmoor updated :angel:

File: pkg/reconciler/v1alpha1/pipeline/resources/dag_test.go [hunk 3]
```diff
@@ -76,12 +76,12 @@ func TestBuild(t *testing.T) {
 			v1alpha1.PipelineSpec{Tasks: []v1alpha1.PipelineTask{a, xDependsOnA, yDependsOnAB, zDependsOnX, b, c}},
 			&DAG{
 				Nodes: map[string]*Node{
-					"a": &Node{Task: a},
-					"b": &Node{Task: b},
-					"c": &Node{Task: c},
-					"x": &Node{Task: xDependsOnA, Prev: []*Node{&Node{Task: a}}},
-					"y": &Node{Task: yDependsOnAB, Prev: []*Node{&Node{Task: b}, &Node{Task: a}}},
-					"z": &Node{Task: zDependsOnX, Prev: []*Node{nodeX}},
```
Comment (bobcatfish) [inline pkg/reconciler/v1alpha1/pipeline/resources/dag_test.go]: oooo pretty :D

**Comment by bobcatfish** [comment]
thanks as always for your attention to detail @vdemeester !
/lgtm

I saw @mattmoor was reviewing too, just in case there's something more to discuss with him, feel free to remove:
/hold

quick question, do we have anything in place to do this automatically? i think with prow it has to be done manually - and @mattmoor has some bots?

**Comment by vdemeester** [comment]
@bobcatfish :stuck_out_tongue_closed_eyes: :wink: 

Yeah, @mattmoor is working on the bots to automate that (in the form of github suggestion in the future) :man_dancing:

**Comment by mattmoor** [comment]
@vdemeester needs a rebase :(

**Comment by vdemeester** [comment]
@mattmoor done :wink:

**Pull request merged**